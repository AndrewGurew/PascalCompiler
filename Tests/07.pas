program test;
var
a: Integer;
		b:String;








c:Integer;





									d:String; 