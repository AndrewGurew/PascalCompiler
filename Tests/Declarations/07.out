(3, 2) - duplicate declaration of b